### ISSUE TITLE
[Bug]: Token issue

### ISSUE THREAD

**Issue by GillesWi** [Brandawg93/homebridge-nest-cam#550]
### Describe the bug

Access blocked: Nest’s request is invalid
<img width="1435" alt="Screenshot 2023-05-12 at 6 06 48 PM" src="https://github.com/Brandawg93/homebridge-nest-cam/assets/77160203/8b9138ba-feac-497f-a041-b3958dd21a73">

### Debug Output

You can’t sign in because Nest sent an invalid request. You can try again later, or contact the [developer](https://github.com/Brandawg93/homebridge-nest-cam/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=%5BBug%5D%3A+) about this issue. [Learn more about this error](https://support.google.com/accounts/answer/12379384)
If you are a developer of Nest, see [error details](https://github.com/Brandawg93/homebridge-nest-cam/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=%5BBug%5D%3A+).
Error 400: invalid_request

### Steps to reproduce

1. Downloading plugin
2. Getting the token

### Device Type

raspberry pi

### iOS Version

/

### Camera Types

Nest hello doorbell V1

### Bug Persistence

Consistently

### Last Working Version

V6.0.3

**Comment by Ralk** [comment]
Where you able to find a workaround for this?

**Comment by GillesWi** [comment]
Yes, you can use an older version that still uses the refresh token method. 
Or even easier, just copy this code below in your Nest cam json config. 
Just change the options to your options. To retrieve the token and cookies you can look in there docs are you can even use the Hoobs sidecar plugin on Google chrome for auto config :)

{
    "googleAuth": {
        "issueToken": "YOUR_TOKEN_HERE",
        "cookies": "YOUR_COOKIES_HERE"
    },
    "options": {
        "ffmpegCodec": "libx264",
        "alertCooldownRate": 180,
    },
    "platform": "Nest-cam"
}
